curated_transformers/models/mpt/causal_lm.py

```diff
@@ -12,15 +12,15 @@
 from ..hf_hub.conversion import state_dict_from_hf, state_dict_to_hf
 from ..output import CausalLMOutputWithCache
 from ..transformer import TransformerCausalLM
-from ._hf import CAUSAL_LM_HF_PARAM_KEY_TRANSFORMS, convert_hf_config
+from ._hf import CAUSAL_LM_HF_PARAM_KEY_TRANSFORMS, _config_from_hf, _config_to_hf
 from .config import MPTConfig
 from .decoder import MPTDecoder
 
 # Only provided as typing.Self in Python 3.11+.
 Self = TypeVar("Self", bound="MPTCausalLM")
 
 
-class MPTCausalLM(TransformerCausalLM[MPTConfig], FromHFHub, Quantizable):
+class MPTCausalLM(TransformerCausalLM[MPTConfig], FromHFHub[MPTConfig], Quantizable):
     """
     `MosaicML MPT`_ causal language model.
```

Comment:
Is the `_` a typo?

Reply:
Nope, that's just weird RST syntax used for inline links.